test/integration/services/data/test-get-reduction-reasons.js

```diff
@@ -31,7 +33,12 @@ describe('services/data/get-reduction-reasons', function () {
   it('should return an array of reductions reasons ref data, including category', function () {
```

Comment:
Add some detail about cms to the test description?

Reply:
updated